README.md

```diff
@@ -37,16 +37,16 @@ Set the following configurations in your settings:
     "projectId": "[Sentry project ID, e.g. "1334031"]",
     "linePatterns": [
         // List of RegExp patterns that match error handling code, e.g. "throw new Error+\\(['\"]([^'\"]+)['\"]\\)",
-        // !! Make sure to capture the error message in a RegExp group !!
+        // !! Make sure to capture the error message in a RegExp group and escape special characters to ensure compatibility with JSON Schema!!
```

Comment:
I'd drop the "Schema" because you need to escape to be compliant with JSON (not with the schema)
The exclamation marks seem very aggressive